### ISSUE TITLE
Fix Various a11y issues on homepage & subpages

### ISSUE THREAD

**Issue by surajgoraya** [ASSETS-Conference/assets2024#79]
As discovered by some members of the community, it seems like we're missing alt-tags in some of our images, as well as have some inaccessible elements such as "on this page" which need to be corrected.
